pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -4411,6 +4411,10 @@ span.gwt-RadioButton label {
   margin-top: -3px;
 }
 
+#contentCleanerPanel .gwt-TextBox {

```

Comment:
IMO it's strange to have specific style for concrete #contentCleanerPanel here in commons gwt modules project.
This element with id "contentCleanerPanel" is created in PUC where also located css styles for different themes. Does it make sense to move this specific style to user-console project?